.github/workflows/release.yml

```
@@ -19,14 +20,16 @@ jobs:
         run: |
           # we expect a tag on the form of: ControllerContainerBlockSSHPort/v0.1.0
           # and want:
+          #   policy_name="ControllerContainerBlockSSHPort"
           #   policy_working_dir="policies/ControllerContainerBlockSSHPort"
           #   policy_version="0.1.0"
           #   policy_id="controller-container-block-ssh-port" # from metadata.yml
-          policy_dir_name=$( echo ${{ github.ref_name }} | sed 's/\(.*\)\/\(.*\)$/\1/' )
-          policy_working_dir=policies/"$policy_dir_name"
+          policy_name=$( echo ${{ github.ref_name }} | sed 's/\(.*\)\/\(.*\)$/\1/' )
+          policy_working_dir=policies/"$policy_name"
           policy_ociUrl=$(yq -r '.annotations."io.kubewarden.policy.ociUrl"' $policy_working_dir/metadata.yml)
```

```suggestion
          policy_ociUrl=$(yq -r '.annotations."io.kubewarden.policy.title"' $policy_working_dir/metadata.yml)
```

I suppose this could be `io.kubewarden.policy.title`, but now I'm wondering why I didn't use that.